content/blogposts/2024/sourcegraph-release-may-2024.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Sourcegraph May 2024 release"
+publishDate: 2024-05-07T10:00-07:00
+description: "The latest Sourcegraph release includes a new point-in-time search filter for Code Search."
+tags: [blog, release]
+slug: "release/May-2024"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/may-2024/sourcegraph-may2024-release-image.png
+changelogItems:
+  - description: "Increased both the performance and the default timeout for language usage insights, which will significantly reduce pain for customers with large repos. Both are configurable using environment variables on the frontend container."
```

Comment:
Can we link to docs for this one, to show users how to configure those variables?